### ISSUE TITLE
Dock crashes after updates

### ISSUE THREAD

**Issue by dusanx** [nwg-piotr/nwg-dock-hyprland#53]
After the latest updates on Artix, using nwg-dock-hyprland 0.2.1-1 and hyprland 0.42.0-1, dock crashes on start with:

```
> nwg-dock-hyprland -x -ico app-launcher -i 40                                                                                                                                                     

INFO[0000] Using auto-detected launcher command: 'nwg-drawer'
INFO[0000] Using style: /home/dusanx/.config/nwg-dock-hyprland/style.css
FATA[0000] Couldn't list clients: json: cannot unmarshal number into Go struct field client.fullscreen of type bool
```

In rare (<10%) of starts it manages to start but crashes few seconds later.

If you want me to test something please let me know.

**Comment by nwg-piotr** [comment]
Hi,

It has been resolved in v0.2.2. I can't update the nwg-dock-hyprland AUR package, as it's already in Arch Extra-Testing. I updated nwg-dock-hyprland-bin, try using it instead temporarily. For now I can't say when all the nwg-shell stuff is going to hit the Extra repo.

**Comment by dusanx** [comment]
Thanks, -bin works well for now!
